apps/crn-server/test/data-providers/contentful/pages.data-provider.test.ts

```diff
@@ -94,4 +94,10 @@ describe('Pages Contentful Data Provider', () => {
       );
     });
   });
+
+  describe('Fetch-by-id', () => {
+    test('should throw an error', async () => {
+      await expect(pageDataProvider.fetchById()).rejects.toThrow();
```

Comment:
same here - basically i think it's best to be more explicit as to why it's throwing